tests/dummy/app/templates/getting-started/assets.hbs

```diff
@@ -1,4 +1,16 @@
 {{! template-lint-disable block-indentation }}
+<h1 id="included-assets">Included Assets</h1>
+
+<p>ember-bootstrap uses the Bootstrap CSS classes while replacing the behaviors from the components Bootstrap implements
+    in <code>bootstrap.js</code>, such as toggle, navbar, modal, etc., with equivalent, CSS class-compatible
+    native Ember components.</p>
+
+<p>If you are not using a preprocessor, ember-bootstrap will import the static CSS files. If you are using a preprocessor
+    like LESS or SASS, ember-bootstrap imports the preprocessor-ready versions of the styles.</p>
+
+<p>ember-bootstrap specifically excludes `bootstrap.js`. The jQuery and Ember ways of control flow and animation sometimes
```

Comment:
Actually thinking about it I would change the paragraph to read:
```<p>We have deliberately excluded `bootstrap.js` from ember-bootstrap, as the jQuery and Ember methods of control flow and animation sometimes don't play well together, causing unpredictable results. This is in fact the main motivation behind ember-bootstrap. It is possible to import `bootstrap.js` from bower_components or the vendor folder but this is NOT recommended or supported and you will be on your own. You have been warned!!!</p>```

This puts the paragraph into the active rather than passive voice, makes clear it is possible to import `bootstrap.js` but strongly advises against it.